src/NServiceBus.Transport.RabbitMQ/Sending/MessageDispatcher.cs

```diff
@@ -68,5 +75,14 @@ Task PublishMessage(MulticastTransportOperation transportOperation, ConfirmsAwar
             return channel.PublishMessage(transportOperation.MessageType, message, properties, cancellationToken);
         }
 
+        void ThrowIfDelayedDeliveryDisabled(IOutgoingTransportOperation transportOperation)
```

Comment:
Maybe adjust the name? Reading the code I though, why validate this for sends/publishes until I peeked the implementation:

```suggestion
        void ThrowIfDelayedAndDelayedDeliveryIsDisabled(IOutgoingTransportOperation transportOperation)
```

Reply:
```suggestion
        void ThrowIfDelayedDeliveryIsDisabledAndMessageIsDelayed(IOutgoingTransportOperation transportOperation)
```